app/src/main/java/org/fossasia/openevent/app/event/detail/EventDetailFragment.java

```diff
@@ -100,9 +103,21 @@ public void onActivityCreated(@Nullable Bundle savedInstanceState) {
     }
 
     @Override
+    public void onAttach(Context context) {
+        super.onAttach(context);
+        if (context instanceof OnEventLoadedListener) {
```

Comment:
This is not the correct way. Interfaces were made so that you don't cast classes. However, as I think we'll be changing this pattern, I'll think of other solution for communication between fragments